src/main/java/org/example/hw3_2/Main.java

```diff
@@ -0,0 +1,27 @@
+package org.example.hw3_2;
+
+import java.util.ArrayList;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        Participant human = new Human("Людина", 100, 2);
+        Participant cat = new Human("Кіт", 50, 1);
+        Participant robot = new Human("Робот", 200, 5);
```

Comment:
the same

Reply:
fix

Reply:
it was not fixed, robot cannot be a human 🙂

Reply:
in the process of

Reply:
fixed